.github/workflows/tekton_task_tests.yaml

```diff
@@ -22,8 +22,26 @@ jobs:
           dir_names: "true"
           dir_names_max_depth: "3"
       - name: Show changed dirs
+        id: show-changed-dirs
+        shell: bash
         run: |
           echo ${{ steps.changed-dirs.outputs.all_changed_files }}
+          trustedArtifactsBasedTasks=()
+          for dir in ${{ steps.changed-dirs.outputs.all_changed_files }}
+          do
+            echo "$dir"
+            baseDir=$(basename "${dir}")
+            taskFile="${dir}/${baseDir}.yaml"
+            if [ -f "${taskFile}" ]; then
+              ociStageParam=$(yq '.spec.params[] | select(.name == "ociStorage") | .name' "${taskFile}")
+              if [ ! -z "${ociStageParam}" ]; then
+                echo "Found a trusted artifacts compatible task: ${taskFile}"
+                trustedArtifactsBasedTasks+=($dir)
+              fi
+            fi
+          done
+          echo "trustedArtifactsBasedTasks: ${trustedArtifactsBasedTasks[@]}"
+          echo "trustedArtifactsBasedTasks=${trustedArtifactsBasedTasks[@]}" >> $GITHUB_OUTPUT
```

Comment:
Just to be clear, we assume that if the task has the `ociStorage` parameter, the tests were also updated to support this, right?

Reply:
correct